docs/write_test/concepts.md

```diff
@@ -1,78 +1,115 @@
-# Concepts
+# Basic concepts
 
-- [Test suite and test case](#test-suite-and-test-case)
+- [Test case and Test suite](#test-case-and-test-suite)
 - [Node and Environment](#node-and-environment)
-- [Platform](#platform)
-- [Tools and Scripts](#tools-and-scripts)
-- [Feature](#feature)
+- [Extension](#extension)
+  - [Platform](#platform)
+  - [Tool and Script](#tool-and-script)
+  - [Feature](#feature)
+  - [Combinator](#combinator)
+    - [Grid combinator](#grid-combinator)
+    - [Batch combinator](#batch-combinator)
+  - [Transformer](#transformer)
 - [Runbook](#runbook)
```

Comment:
The runbook is an important concept for run test cases too.

Reply:
Added in runbook.md